### ISSUE TITLE
NTR: Prenatal ultrasound: snowflake effect in amniotic cavity (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8693]
New term request Prenatal ultrasound: snowflake effect in amniotic cavity (elthat_160504162520): Echogenic particles floating

**Comment by pnrobinson** [comment]
@tsparks @MickeySegal

I think this finding needs to be specified as to the gestational age?

**Comment by MickeySegal** [comment]
For Harlequin ichthyosis and Aplasia cutis congenita we have from Benacerraf (2008) Ultrasound of Fetal Syndromes "Snowflake effect - pieces of skin float in the amniotic fluid when skin is denuded in utero".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000341
- label: Snowflake effect in amniotic cavity
- parents: [Abnormality of the amniotic fluid (HP:0001560)](https://hpo.jax.org/app/browse/term/HP:0001560)
- pmid: PMID:33569485
- def: Dense floating particles in amniotic fluid, said to resemble snowflakes. This finding can result from shedding of the epidermis in conditions such as Harlequin ichthyosis.
- comment: See Figure 1A of PMID:33569485.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8693
- synonyms: Snowflake sign(NONE)
